packages/slinkity/cli/index.js

```diff
@@ -65,9 +66,10 @@ const toEleventyOptions = (allOptions) => {
   for (let [name, value] of Object.entries(allOptions)) {
     if (eleventyArgs[name] != null) {
       eleventyOptions[name] = value
-    }
-    if (name === 'serve' && value === true) {
+    } else if (name === 'serve' && value === true) {
       eleventyOptions.watch = true
+    } else if (name === 'port') {
+      eleventyOptions[name] = value
```

Comment:
Ah, I see what's going on here. I think I should explain the diff between `slinkityArgs` and `eleventyArgs` briefly:
- `eleventyArgs` are flags passed directly to our eleventy config, unchanged
- `slinkityArgs` are flags that we intercept and process ourselves

There was a time when we managed our own dev server, so we needed to intercept `--port` to be handled separately. This is no longer the case! For a more sustainable solution, I'd suggest moving the `port: {...}` config above _out_ of the `slinkityArgs` object to the `eleventyArgs` object. This avoids the need for a new if statement here.